Makefile

```diff
@@ -19,26 +19,27 @@ else
 QUAY_TAG_EXPIRATION=never
 endif
 
+ROX_PRODUCT_BRANDING ?= STACKROX_BRANDING
+
 # ROX_IMAGE_FLAVOR is an ARG used in Dockerfiles that defines the default registries for main, scaner, and collector images.
-# ROX_IMAGE_FLAVOR valid values are: development_build, stackrox.io, rhacs.
-# The value is assigned as following:
-# 1. Use environment variable if provided.
-# 2. If makefile variable GOTAGS is contains "release", use "stackrox.io".
-# 3. Otherwise set it to "development_build" by default, e.g. for developers running the Makefile locally.
-ROX_IMAGE_FLAVOR ?= $(shell if [[ "$(GOTAGS)" == *"$(RELEASE_GOTAGS)"* ]]; then echo "stackrox.io"; else echo "development_build"; fi)
+# ROX_IMAGE_FLAVOR valid values are: development_build, stackrox.io, rhacs, opensource.
+ROX_IMAGE_FLAVOR ?= $(shell \
+	if [[ "$(ROX_PRODUCT_BRANDING)" == "STACKROX_BRANDING" ]]; then \
+	  echo "opensource"; \
+	elif [[ "$(GOTAGS)" == *"$(RELEASE_GOTAGS)"* ]]; then \
+	  echo "stackrox.io"; \
+	else \
+	  echo "development_build"; \
+	fi)
+
+BUILD_IMAGE := quay.io/stackrox-io/apollo-ci:$(shell sed 's/\s*\#.*//' BUILD_IMAGE_VERSION)
 
-ROX_PRODUCT_BRANDING ?= STACKROX_BRANDING
 DEFAULT_IMAGE_REGISTRY := quay.io/stackrox-io
-BUILD_IMAGE_VERSION=$(shell sed 's/\s*\#.*//' BUILD_IMAGE_VERSION)
-BUILD_IMAGE := $(DEFAULT_IMAGE_REGISTRY)/apollo-ci:$(BUILD_IMAGE_VERSION)
-DOCS_IMAGE_BASE := $(DEFAULT_IMAGE_REGISTRY)/docs
-
-ifdef CI
-    CI_QUAY_REPO := rhacs-eng
-    DOCS_IMAGE_BASE := quay.io/$(CI_QUAY_REPO)/docs
+ifneq ($(ROX_PRODUCT_BRANDING),STACKROX_BRANDING)
+	DEFAULT_IMAGE_REGISTRY := quay.io/rhacs-eng
```

Comment:
Wouldn't it be better to default to open source branding? so `ifeq($(ROX_PRODUCT_BRANDING),RHACS_BRANDING)` instead